libs/blocks/locui/loc/index.js

```diff
@@ -97,12 +97,20 @@ async function loadProjectSettings(projSettings) {
   }
 }
 
+function sanitize(url) {
+  const hlxUrl = /^https?:\/\/[^/?#]+/g;
+  const [hostname] = url.match(hlxUrl);
+  const enDash = hostname.replaceAll('–', '--');
+  const newURL = url.replace(hlxUrl, enDash);
+  return new URL(newURL);
+}
+
 async function loadDetails() {
   setStatus('details', 'info', 'Loading languages and URLs.');
   try {
     const resp = await fetch(previewPath);
     const json = await resp.json();
-    const jsonUrls = json.urls.data.map((item) => new URL(item.URL));
+    const jsonUrls = json.urls.data.map(({ URL }) => sanitize(URL));
```

Comment:
Has the data structure also changed? Previously `item.URL` was being sent, now it's just `URL`, which corresponds to the previous `item`.

Reply:
I simply thought spreading this to get the property name would look better, but dont want to introduce a new pattern so I have reverted this.

Reply:
I 100% missed the extra curly braces, my bad 😅